README.md

```diff
@@ -67,6 +67,17 @@ Therefore it is recommended to clone the training repository in the Linux filesy
 > installing packages from GitHub, resulting in failures to update sensei
 > correctly.
 
+⚠️ On Mac M1 or M2, you need to follow special steps to enable PDF generation.
+You must do this before the `npm install`, otherwise you will have an unused version of chromium installed in `~/.cache/puppeteer`
```

Comment:
```suggestion
You must do this before the `npm install`, otherwise you will have an unused version of chromium installed in `~/.cache/puppeteer`.
```